### PR TITLE
Test gap for lldp APPL_DB data consistency

### PR DESCRIPTION
**Pull request by ZhaohuiS**
### Description of PR

Summary:
Fixes # (issue)
Address https://github.com/sonic-net/sonic-mgmt/issues/13937

ADO:28941599
### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [x] 202311
- [x] 202405

#### What is the motivation for this PR?
To address https://github.com/sonic-net/sonic-mgmt/issues/13937
Test plan: https://github.com/sonic-net/sonic-mgmt/pull/14080

#### How did you do it?
Add a new test script to test  lldp APPL_DB data consistency
#### How did you verify/test it?
run lldp/test_lldp_syncd.py

### CODE REVIEW

**Comment by yutongzhang-microsoft** [comment]
Hi, @ZhaohuiS , can you help me confirm if this new added test script can run in PR test?

File: tests/lldp/test_lldp_syncd.py [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+import json
+from tests.common.helpers.sonic_db import SonicDbCli
+import logging
+import time
+from tests.common.reboot import reboot, REBOOT_TYPE_COLD
+
+logger = logging.getLogger(__name__)
+
+APPL_DB = "APPL_DB"
+
```
[the remark this answers is not in the document]
Comment (ZhaohuiS) [reply tests/lldp/test_lldp_syncd.py:14]: Done

**Comment by ZhaohuiS** [comment]
> Hi, @ZhaohuiS , can you help me confirm if this new added test script can run in PR test?

@yutongzhang-microsoft It should be ran on kvm testbed.

**Review by yutongzhang-microsoft — APPROVED** [review]

**Review by yejianquan — APPROVED** [review]
LGTM

**Comment by yejianquan** [comment]
@ZhaohuiS do we need it in 202405 branch?

**Comment by bingwang-ms** [comment]
@ZhaohuiS The test case is failing with error .
```
  File "/var/src/sonic-mgmt_vms2-4-t0-2700_646f1405735219c3e54440f2/tests/lldp/test_lldp_syncd.py", line 36, in get_lldp_entry_content
    return db.hget_all("LLDP_ENTRY_TABLE:{}".format(interface))
AttributeError: 'SonicDbCli' object has no attribute 'hget_all'
```

Is there any dependent PR not merged?

**Comment by bingwang-ms** [comment]
Issue fixed in PR https://github.com/sonic-net/sonic-mgmt/pull/14846

**Pull request merged**